packages/dd-trace/test/plugins/externals.json

```diff
@@ -34,6 +34,10 @@
     }
   ],
   "jest": [
+    {
+      "name": "jest",
```

Comment:
Does `jest` depend on `jest`? Not sure I understand this external dependency.

Reply:
this is just to be able to run jest tests within the test suite